### PR TITLE
Add conda recipe for bmipy package

### PR DESCRIPTION
**Pull request by mcflugen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mcflugen** [comment]
@conda-forge-admin please rerender

**Comment by mcflugen** [comment]
@conda-forge/help-python This should be good to go now (finally). Thanks!

File: recipes/bmipy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  run:
+    - python
+    - black  # [py3k]
+    - click
+    - jinja2
+    - numpy
+    - typing  # [py2k]
+
+test:
+  requires:
```
Comment (chrisburr) [inline recipes/bmipy/meta.yaml]: It looks like you don’t need pytest here?

Comment (mcflugen) [reply recipes/bmipy/meta.yaml]: Correct. Thank you!

**Comment by mcflugen** [comment]
@chrisburr thank you for your help! Having the unnecessary pytest requirement, hid a missing run requirement.

**Pull request merged**